client/src/components/Account/Friends/FriendsTab.tsx

```diff
@@ -103,7 +103,7 @@ function FriendsTab() {
           );
         })
       ) : (
-        <div>still rendering</div>
+        <Spinner aria-label="Extra small spinner example" size="xl" />
```

Comment:
I think this `aria-label` is the default one that comes with Flowbite. Can you change it to something more semantic like `friends loading spinner` or something similar?